packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerCache.java

```diff
@@ -114,51 +109,42 @@ static void saveResult(
     editor.apply();
   }
 
-  static void clear() {
-    if (getFilePref == null) {
-      return;
-    }
-    getFilePref.edit().clear().apply();
+  void clear() {
+    prefs.edit().clear().apply();
   }
 
-  static Map<String, Object> getCacheMap() {
-    if (getFilePref == null) {
-      return new HashMap<>();
-    }
+  Map<String, Object> getCacheMap() {
+
     Map<String, Object> resultMap = new HashMap<>();
-    Boolean hasData = false;
+    boolean hasData = false;
 
-    if (getFilePref.contains(FLUTTER_IMAGE_PICKER_IMAGE_PATH_KEY)) {
-      resultMap.put(MAP_KEY_PATH, getFilePref.getString(FLUTTER_IMAGE_PICKER_IMAGE_PATH_KEY, ""));
+    if (prefs.contains(FLUTTER_IMAGE_PICKER_IMAGE_PATH_KEY)) {
+      final String imagePathValue = prefs.getString(FLUTTER_IMAGE_PICKER_IMAGE_PATH_KEY, "");
+      resultMap.put(MAP_KEY_PATH, imagePathValue);
       hasData = true;
     }
 
-    if (getFilePref.contains(SHARED_PREFERENCE_ERROR_CODE_KEY)) {
-      resultMap.put(
-          MAP_KEY_ERROR_CODE, getFilePref.getString(SHARED_PREFERENCE_ERROR_CODE_KEY, ""));
+    if (prefs.contains(SHARED_PREFERENCE_ERROR_CODE_KEY)) {
+      final String errorCodeValue = prefs.getString(SHARED_PREFERENCE_ERROR_CODE_KEY, "");
+      resultMap.put(MAP_KEY_ERROR_CODE, errorCodeValue);
       hasData = true;
-      if (getFilePref.contains(SHARED_PREFERENCE_ERROR_MESSAGE_KEY)) {
-        resultMap.put(
-            MAP_KEY_ERROR_MESSAGE, getFilePref.getString(SHARED_PREFERENCE_ERROR_MESSAGE_KEY, ""));
+      if (prefs.contains(SHARED_PREFERENCE_ERROR_MESSAGE_KEY)) {
+        final String errorMessageValue = prefs.getString(SHARED_PREFERENCE_ERROR_MESSAGE_KEY, "");
+        resultMap.put(MAP_KEY_ERROR_MESSAGE, errorMessageValue);
       }
     }
 
     if (hasData) {
-      if (getFilePref.contains(SHARED_PREFERENCE_TYPE_KEY)) {
-        resultMap.put(MAP_KEY_TYPE, getFilePref.getString(SHARED_PREFERENCE_TYPE_KEY, ""));
+      if (prefs.contains(SHARED_PREFERENCE_TYPE_KEY)) {
+        final String typeValue = prefs.getString(SHARED_PREFERENCE_TYPE_KEY, "");
+        resultMap.put(MAP_KEY_TYPE, typeValue);
       }
 
-      if (getFilePref.contains(SHARED_PREFERENCE_MAX_WIDTH_KEY)) {
-        resultMap.put(
-            MAP_KEY_MAX_WIDTH,
-            Double.longBitsToDouble(getFilePref.getLong(SHARED_PREFERENCE_MAX_WIDTH_KEY, 0)));
-      }
+      final long maxWidthValue = prefs.getLong(SHARED_PREFERENCE_MAX_WIDTH_KEY, 0);
```

Comment:
Why did we remove the `contains` check? Is it not necessary for long values?

Reply:
I forgot to remove it from my first PR. reverted.